examples/gnrc_minimal/Makefile

```diff
@@ -13,7 +13,7 @@ BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h
 #USEMODULE += xbee
 
 ## set default UART to use in case none was defined
-#XBEE_UART ?= "UART_NUMOF-1"
+#XBEE_UART ?= "1"
```

Comment:
It gets undefined, and won't work. For now a quick test make the board hang without any error, same for `UART_NUMOF-1`.